### PR TITLE
NE Apollyon Rework

### PR DESCRIPTION
**Pull request by jmcmorris**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Migrate and rework NE Apollyon Limbus area to be in the new battlefield framework. All of this work is heavily based off of the following captures and sources:
https://www.youtube.com/watch?v=CNJNBhTBse8
https://www.youtube.com/watch?v=CoD5OEywDOI
https://ffxiclopedia.fandom.com/wiki/NE_Apollyon
https://www.bg-wiki.com/ffxi/North_East_Apollyon

## Steps to test these changes

```
!addkeyitem black_card
!addkeyitem cosmo_cleanse
!pos 600 -0.5 -600 38
```

Play through the area following the above guides and captures.

### CODE REVIEW

**Review by WinterSolstice8 — APPROVED** [review]

File: scripts/battlefields/Apollyon/ne_apollyon.lua [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+local content = Limbus:new({
+    zoneId           = xi.zone.APOLLYON,
+    battlefieldId    = xi.battlefield.id.NE_APOLLYON,
+    maxPlayers       = 18,
+    timeLimit        = utils.minutes(30),
+    index            = 3,
+    area             = 4,
+    entryNpc         = '_12i',
+    requiredKeyItems = { xi.ki.COSMO_CLEANSE, xi.ki.BLACK_CARD, message = ID.text.YOU_INSERT_THE_CARD_POLISHED },
+    lossEventParams  = { [5] = 1 },
```
Comment (cocosolos) [inline scripts/battlefields/Apollyon/ne_apollyon.lua:24]: I must've missed this on the SE rework, but does this only apply to the loss event? Is the winning CS still spitting you out on the west side?

Comment (jmcmorris) [reply scripts/battlefields/Apollyon/ne_apollyon.lua:24]: Good question! Double checking this.

Comment (jmcmorris) [reply scripts/battlefields/Apollyon/ne_apollyon.lua:24]: Good catch! I looked into this and added in a fix for this area and will PR a fix for SE Apollyon soon.

**Comment by jmcmorris** [comment]
CI failure is due to a bad require in one of the NW Apollyon files that has been deleted in that PR. So once that PR is merged I'll rebase and it'll pass CI then.

**Review by claywar — APPROVED** [review]

**Comment by jmcmorris** [comment]
Fixed https://github.com/LandSandBoat/server/issues/881

**Comment by jmcmorris** [comment]
Fixed https://github.com/LandSandBoat/server/issues/879 as well

**Pull request merged**